grpc/estimator_service.h

```diff
@@ -12,108 +12,92 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// An implementation of the `Estimator` gRPC service.
+// An implementation of the `Batch` gRPC service.
 
-#ifndef GRPC_ESTIMATOR_SERVICE_H
-#define GRPC_ESTIMATOR_SERVICE_H
-
-#include <memory>
-#include <vector>
+#ifndef GRPC_BATCH_SERVICE_H
+#define GRPC_BATCH_SERVICE_H
 
 #include <grpcpp/server_context.h>
 #include <grpcpp/support/status.h>
 #include <mujoco/mujoco.h>
+
```

Comment:
These includes (memory, vector) should be first.

Reply:
Fixed.